### PR TITLE
Migrate icons to SVG

### PR DESCRIPTION
**Pull request by anselmbradford**
`cf-icon` references need to be removed in order to merge https://github.com/cfpb/capital-framework/pull/906. This converts the icons to SVG.

## Changes

- Convert icon fonts to SVG.

## Testing

- Compare this branch to production. Icons and drop-down should show up on:
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/feedback/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/test/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/feedback/
 - http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/

## Note

- ~~Contains https://github.com/cfpb/college-costs/pull/346. I'll rebase when that goes in.~~

### CODE REVIEW

File: src/disclosures/css/cf-enhancements.less [hunk 1]
```diff
@@ -350,8 +350,7 @@ a.btn__full-small.btn__link {
         border-left: 1px solid @gray-40;
         background-color: @gray-10;
         color: @gray-60;
-        content: @cf-icon-down;
-        font-family: 'CFPB Minicons';
+        content: background-image: url( '../icons/down.svg' );
```
Comment (anselmbradford) [inline src/disclosures/css/cf-enhancements.less]: I'm not sure if this is the proper path to resolve this on production?

Comment (anselmbradford) [reply src/disclosures/css/cf-enhancements.less]: Seems to work when I move this into develop-apps

**Review by higs4281 — APPROVED** [review]
The only difference I see is the weight of the down-pointing chevron for changing program length on the disclosure's first page.

**Comment by anselmbradford** [comment]
> The only difference I see is the weight of the down-pointing chevron for changing program length on the disclosure's first page.

:+1: we can proceed with this, the chevron is lighter weight in the SVG CF provides.

**Pull request merged**